src/Components/Register/registerValidations.js

```diff
@@ -0,0 +1,72 @@
+// Config
+import config from '../../CritCore/Config/config';
+
+// Translations
+import getTranslations from '../../CritCore/Translations/Translations';
+
+const { language }         = config,
+translations               = getTranslations(language),
+registerInputErrors        = translations.register,
+msgErrorInput              = registerInputErrors.errors,
+msgErrorInputEmail         = msgErrorInput.email,
+msgErrorInputUserName      = msgErrorInput.userName,
+msgErrorInputPassword      = msgErrorInput.password,
+msgErrorInputAlterEgo      = msgErrorInput.alterEgo,
+msgErrorInputIsPwdMatched  = msgErrorInput.isPwdMatched,
+msgErrorInputFieldRequired = msgErrorInput.fieldRequired;
+
+const registerValidator = {
+  email: ({ value }) => {
+    //eslint-disable-next-line
+    const emailRegexp =  /^\w+([\.\+\-]?\w+)*@([\.-]?\w+)*(\.\w{2,4})+$/;
+    if (!value && value !== 0){
+      return msgErrorInputFieldRequired;
+    }
+    if (!value.match(emailRegexp)){
+      return msgErrorInputEmail;
+    }
+    return null;
+  },
+  userName: ({ value }) => {
+    const userNameRegexp = /^[a-z\d_]{6,20}$/i;
```

Comment:
Try setting this min/max values from the config or just remove it from the regex

Reply:
I Can't remove min-max without breaking user validation scheme.